File: Student.cpp
#include<iostream>
using namespace std;
class Student
{
    public:
    string fullName;
    int rollNum;
    double semPerentage;
    string collegeName;
    int collegeCode;
    Student()
    {
    cout<<"Student class initialized!!\nIam a student\n";
    }
    ~Student(){}
};
int main()
{
    Student bhoomi;
    cout<<"Enter your name : ";
    cin>>bhoomi.fullName;
    cout<<"Enter your roll number : ";
    cin>>bhoomi.rollNum;
    cout<<"Enter your sem percentage : ";
    cin>>bhoomi.semPerentage;
    cout<<"Enter your college name : ";
    cin>>bhoomi.collegeName;
    cout<<"Enter your college code : ";
    cin>>bhoomi.collegeCode;
    cout<<"Name : "<<bhoomi.fullName<<endl;
    cout<<"RollNum : "<<bhoomi.rollNum<<endl;
    cout<<"SemPercentage : "<<bhoomi.semPerentage<<endl;
    cout<<"CollegeName : "<<bhoomi.collegeName<<endl;
    cout<<"CollegeCode : "<<bhoomi.collegeCode<<endl;
    return 0;
}

File: week4/Student.java
import java.util.Scanner;
class student018
{
public
String fullName;
int rollNum;
double semPercentage;
String collegeName;
int collegeCode;
student018()
{
System.out.println("Student class is created successfully!!\nIam a student\n");
}
protected void finalize() throws Throwable
{
System.out.println("The class is finalized\n");
}
}

public class student
{
public static void main(String[] args)
{
student018 Bhoomika=new student018();;
Scanner input=new Scanner(System.in);
System.out.print("Enter your sweet name : ");
Bhoomika.fullName=input.nextLine();
System.out.print("Enter your rollNumber : ");
Bhoomika.rollNum=input.nextInt();
System.out.print("Enter your semPercentage : ");
Bhoomika.semPercentage=input.nextDouble();
input.nextLine();
System.out.print("Enter your collegeName : ");
Bhoomika.collegeName=input.nextLine();
System.out.print("Enter your collegeCode : ");
Bhoomika.collegeCode=input.nextInt();
System.out.print("Name : "+Bhoomika.fullName+"\n");
System.out.print("RollNumber : "+Bhoomika.rollNum+"\n");
System.out.print("SemPercentage : "+Bhoomika.semPercentage+"\n");
System.out.print("CollegeName : "+Bhoomika.collegeName+"\n");
System.out.print("CollegeCode : "+Bhoomika.collegeCode+"\n");
input.close();
}
}
